rules-tests/Php70/Rector/StaticCall/StaticCallOnNonStaticToInstanceCallRector/Fixture/skip_to_jump_over_parent_method.php.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rector\Tests\Php70\Rector\StaticCall\StaticCallOnNonStaticToInstanceCallRector\Fixture;
+
+class SkipParentMethodCallA {
```

Comment:
static call by class name is not overriding, so the existing `new` is ok, to make it valid, the called class must be abstract:

```suggestion
abstract class SkipParentMethodCallA {
```

see https://3v4l.org/rHmJj vs https://3v4l.org/pq3mF